pythonforandroid/bootstraps/sdl2/build/build.py

```diff
@@ -355,20 +355,29 @@ def make_package(args):
                                   key=LooseVersion)
     build_tools_version = build_tools_versions[-1]
 
-    render(
-        'AndroidManifest.tmpl.xml',
-        'src/main/AndroidManifest.xml',
-        args=args,
-        service=service,
-        service_names=service_names,
-        android_api=android_api,
-        url_scheme=url_scheme)
+    manifest_path = join('src', 'main', 'AndroidManifest.xml'
+
+    if args.load_manifest:
+        shutil.copy(args.load_manifest, manifest_path)
+
+    else:
+        render(
+            'AndroidManifest.tmpl.xml',
+            manifest_path,
+            args=args,
+            service=service,
+            service_names=service_names,
+            android_api=android_api,
+            url_scheme=url_scheme)
+
+        if args.save_manifest:
```

Comment:
I think that save_manifest should happen regardless of whether load_manifest was set. Even if that isn't useful (i.e. saving the one the user provided in the first place), at least it's consistent. Or they could be marked as incompatible arguments instead.

Reply:
You mean make the render even if we overwrite it just after (moving the shutil.copy part after the else branch) ? that seems a bit wasteful, and i'm not sure how it helps with consistency, but i'm fine with it if you prefer.